streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -387,6 +387,15 @@
             + "Note when windowed serde class is used, one needs to set the inner serde class that implements the <code>org.apache.kafka.common.serialization.Serde</code> interface via '"
             + DEFAULT_WINDOWED_KEY_SERDE_INNER_CLASS + "' or '" + DEFAULT_WINDOWED_VALUE_SERDE_INNER_CLASS + "' as well";
 
+
+    public static final String DEFAULT_LIST_KEY_SERDE_INNER_CLASS = CommonClientConfigs.DEFAULT_LIST_KEY_SERDE_INNER_CLASS;
```

Comment:
I am wondering now, why we actually need `CommonClientConfigs.DEFAULT_LIST_KEY_SERDE_INNER_CLASS` (maybe there was a reason by I forgot) -- can't we add the "SERDE" configs only to `StreamsConfig`?